.eslintrc.js

```diff
@@ -93,5 +93,10 @@ module.exports = {
     'react/jsx-uses-vars': 'error',
 
     'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx'] }],
+
+    /**
+     * Some passport properties are only exposed via underscore names
+     */
+    'no-underscore-dangle': 'off',
```

Comment:
What's the advantage/disadvantage of this?

Reply:
I just needed it to get eslint to pass, since I (currently) have to access the logout function I need via `passport._strategy` and our rules don't allow names with leading `_`.  I might be able to fix this later so I don't need to do this.

Reply:
In general, anything named `_name` is a "private" variable, and you aren't usually supposed to access it.